server/src/game/util.js

```diff
@@ -35,4 +35,75 @@ const getRolePool = (config) => {
   return { roles: pool, order };
 };
 
-module.exports = { getDialogue, getRoleData, getRolePool };
+/**
+ * Returns whether or not the given role should act but doesn't require player interaction
+ * @param {String} role 
+ */
+const isEffectivelyPassive = (role) => {
+  switch(role) {
+    case roles.jake:
+    case roles.austin:
+      return true;
+    default:
+      return false
+  }
+};
+
+/**
+ * @param {Player} player 
+ * @return {boolean} If the player is a Josh whose passive should apply
+ */
+const validJosh = (player) => player.role === roles.josh && !player.asleep;
+
```

Comment:
So, josh can only be valid if sydney didn't get him. I love it 😍

Reply:
Yep! If Sydney got him, he's basically a glorified villager haha